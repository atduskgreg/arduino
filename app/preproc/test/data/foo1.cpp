int encoder = 4;
unsigned long speed = 0;

void setup() 
{ 
  pinMode(encoder, INPUT);
}

void loop() 
{ 
    speed = getspeed();
    delay(1000);  
} 

unsigned long getspeed()
{
  unsigned long duration = 0;
  duration = pulseIn(encoder, HIGH);
  return(duration);
} 

