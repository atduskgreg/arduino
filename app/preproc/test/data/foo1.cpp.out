void setup();
void loop();
unsigned long getspeed();
